backend/src/main/java/com/woowacourse/kkogkkog/exception/GlobalControllerAdvice.java

```diff
@@ -35,6 +40,12 @@ public ResponseEntity<ErrorResponse> handleException(ForbiddenException e) {
         return new ResponseEntity<>(new ErrorResponse(e.getMessage()), HttpStatus.FORBIDDEN);
     }
 
+    @ExceptionHandler
+    public ResponseEntity<ErrorResponse> handleException(MethodArgumentNotValidException e) {
+        String message = String.format(INVALID_REQUEST_EXCEPTION_MESSAGE_FORMAT, Objects.requireNonNull(e.getBindingResult().getFieldError()).getDefaultMessage());
+        return handleException(new InvalidRequestException(message));
+    }
+
```

Comment:
아래처럼 `getFieldError` 대신 `getFieldErrors`를 받으면 복수의 필드에 대한 validation 위배 결과를 묶을 수 있습니다. :)

```suggestion
    @ExceptionHandler
    public ResponseEntity<ErrorResponse> handleInvalidRequest(MethodArgumentNotValidException bindingResult) {
        String causes = bindingResult.getFieldErrors()
                .stream()
                .map(DefaultMessageSourceResolvable::getDefaultMessage)
                .collect(Collectors.joining(", "));
        String errorMessage = String.format("잘못된 입력입니다: [%s]", causes);
        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(new ErrorResponse(errorMessage));
    }

```

<img width="553" alt="image" src="https://user-images.githubusercontent.com/73531614/179888238-c7f5fb31-19ab-457b-be69-56daff04007e.png">

Reply:
적용하였습니다.